src/components/services/HomeAssistant.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <Generic :item="item">
+    <template #content>
+      <p class="title is-4">{{ item.name }}</p>
+      <p class="subtitle is-6">
+        <template v-if="item.subtitle">
+          {{ item.subtitle }}
+        </template>
+        <template v-else>
+          {{ details }}
+        </template>
+      </p>
+    </template>
+    <template #indicator>
+      <div v-if="status" class="status" :class="status">
+        {{ status }}
+      </div>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+
+export default {
+  name: "HomeAssistant",
+  mixins: [service],
+  props: {
+    item: Object,
+  },
+  components: {
+    Generic,
+  },
+  data: () => ({
+    status: "",
+    msg: "",
+    version: "",
+    entities: 0,
+  }),
+  computed: {
+    headers: function () {
+      return {
+        'Authorization': `Bearer ${this.item.apikey}`,
+        'Content-Type': 'application/json',
+      };
+    },
+    details: function () {
+      const details = [];
+      const items = this.item.items;
+      const separator = this.item.separator;
+
+      for (const i in items) {
+        const key = items[i];
+
+        switch (key) {
+          case "version":
+            details.push(
+              `v${this.version}`,
+            );
+            break;
+          case "name":
+            details.push(
+              `${this.location_name}`,
+            );
+            break;
+          case "entities":
+            details.push(
+              `Entities: ${this.entities}`,
+            );
+            break;
+          default:
+            details.push(
+              `undefined key ${key} `,
+            );
+        }
+      }
+      
+      return details.join(separator);
+    },
+  },
+  created() {
+    this.fetchServerStatus();
+
+    if (!this.item.subtitle && this.status !== "dead") {
+      if (!this.item.items) this.item.items = ["name", "version"];
+      if (!this.item.separator) this.item.separator = " ";
+      if (!this.item.logo) this.item.logo = `${this.item.url}/static/icons/favicon-192x192.png`;
+      this.fetchServerStats();
+    }
+  },
+  methods: {
+    fetchServerStatus: async function () {
+      const headers = this.headers;
+
+      this.fetch("/api/", { headers })
```

Comment:
You could return the fetch call to make possible to await this.
```suggestion
      return this.fetch("/api/", { headers })
```